src/main/java/net/neoforged/neoforge/registries/RegisterEvent.java

```diff
@@ -13,29 +13,24 @@
 import net.neoforged.bus.api.Event;
 import net.neoforged.fml.ModLoadingContext;
 import net.neoforged.fml.event.IModBusEvent;
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
 
 /**
- * This event fires for each forge and vanilla registry when all registries are ready to have modded objects registered.
- * <p>
- * Fired on the {@link IModBusEvent mod bus}.
+ * Fired for each registry when it is ready to have modded objects registered.
+ * This event is fired for all registries, including registries in {@link net.minecraft.core.registries.BuiltInRegistries}.
```

Comment:
All registries, meaning both built-in and datapack registries? If so, datapack registries should be mentioned here as well.

Reply:
fixed now